pkg/generator/templates.go

```diff
@@ -270,13 +270,13 @@ required = [
 
 [[override]]
   name = "k8s.io/apimachinery"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "302974c03f7e50f16561ba237db776ab93594ef6"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "103fd098999dc9c0c88536f5c9ad2e5da39373ae"
 
 [[override]]
   name = "k8s.io/client-go"
-  # revision for tag "kubernetes-1.10.1"
-  revision = "989be4278f353e42f26c416c53757d16fcff77db"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "1f13a808da65775f22cbf47862c4e5898d8f4ca1"
 
 [[override]]
   name = "sigs.k8s.io/controller-runtime"
```

Comment:
Can you also update the controller-runtime version below to `version = "v0.1.2"`.

Reply:
Turns out [v0.1.3](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.1.3) is out. We can bump up to that since it comes with the leader election pkg.